content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -6,43 +6,40 @@ weight: 10
 
 ## User facing changes
 - ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis).
-- ***Added*** the new [Workload Entry](https://preliminary.istio.io/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
-- ***Added*** support for configuring Gateway topology settings like number of trusted proxies deployed in front
-- ***Added*** experimental mesh-wide tracing configuration API. This API provides control of trace sampling rates, the maximum tag lengths for URL tags, and custom tags extraction for all traces within the mesh. 
-- ***Added*** experimental request classification filter support. This feature enables operators to configure new attributes for use in telemetry, based on request information. A primary use case for this feature is labeling of traffic by API method.  TODO: doc
-- ***Added*** Wasm extension distribution support. TODO: docs
+- ***Added*** the new [Workload Entry](/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- **Added** configuration for gateway topology.
+- ***Added*** experimental [mesh-wide tracing configuration API](docs/tasks/observability/distributed-tracing/configurability/). This API provides control of trace sampling rates, the maximum tag lengths for URL tags, and custom tags extraction for all traces within the mesh. 
```

Comment:
this link requires a leading / for the mesh-wide tracing configuration API.

Reply:
done